### ISSUE TITLE
[a11y]: Autoscrolling carousel on https://168.motc.gov.tw/en

### ISSUE THREAD

**Issue by komali2** [twa11y/twa11y#20]
### What is the accessibility issue?

168.motc.gov.tw's english website has an autoscrolling carousel at the bottom of the page with various infographs about taiwan traffic

this is an accessibility issue: https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#qr-time-limits-pause

Any content that moves in someway should have a way to pause the movement. Ideally there's no reason to autoscroll this carousel, so it shouldn't autoscroll. 

### Please add a screenshot here (if applicable)

![Uploading Screenshot from 2022-03-09 16-18-51.png…]()

### How to reproduce the issue?

1. Go to this page: https://168.motc.gov.tw/en
2. scroll partway down page
3. wait and watch carousel autoscroll
